### PR TITLE
Hinh

### PR DESCRIPTION
**Pull request by daokhanhhuyen2308**

### CODE REVIEW

**Comment by daokhanhhuyen2308** [comment]
ok

**Comment by daokhanhhuyen2308** [comment]
ok

File: src/main/java/com/t3h/e_commerce/configuration/WebSecurityConfig.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public SecurityFilterChain filter(HttpSecurity httpSecurity) throws Exception {
         httpSecurity.cors(AbstractHttpConfigurer::disable)
                 .csrf(AbstractHttpConfigurer::disable);
         httpSecurity.authorizeHttpRequests(auth -> auth.requestMatchers(Endpoints.Public_Endpoints).permitAll()
-                        .requestMatchers("/guests/**").permitAll()
```
Comment (rojectHinh14) [inline src/main/java/com/t3h/e_commerce/configuration/WebSecurityConfig.java]: ok

File: src/main/resources/templates/cms/common/footer.html [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+<footer>
```
Comment (rojectHinh14) [inline src/main/resources/templates/cms/common/footer.html:1]: oke

**Comment by thanggw** [comment]
okkkkkk chi

**Comment by thanggw** [comment]
comment +1 okeeee

**Comment by daokhanhhuyen2308** [comment]
ok

**Comment by daokhanhhuyen2308** [comment]
ok

**Pull request merged**